velox/common/Enums.h

```diff
@@ -22,14 +22,14 @@ namespace facebook::velox {
 
 struct Enums {
   /// Helper function to invert a mapping from enum type to name.
-  template <typename E>
-  static folly::F14FastMap<std::string, E> invertMap(
-      const folly::F14FastMap<E, std::string>& mapping) {
-    folly::F14FastMap<std::string, E> inverted;
+  template <typename EnumType, typename StringType>
+  static auto invertMap(
+      const folly::F14FastMap<EnumType, StringType>& mapping) {
+    folly::F14FastMap<StringType, EnumType> inverted;
```

Comment:
Maybe invert always should be string_view, because even in case when `kNames` `StringType` is string, we don't need to have string in `kValues`
```suggestion
  template <typename EnumType, typename StringType>
  static auto invertMap(
      const folly::F14FastMap<EnumType, StringType>& mapping) {
    folly::F14FastMap<std::string_view, EnumType> inverted;
```